test/helpers.go

```diff
@@ -0,0 +1,94 @@
+package test
```

Comment:
Skip is fine I guess.  I just find it kind of brittle since it needs updated based on something outside the code.  Can we not run the test on RHEL too? Should we be checking `/etc/redhat-release` (which is a symlink to `fedora-release` on fedora, well f26 at least, don't think I ever checked any other version)?  

I think this gives us some benefit either way we go so I wouldn't complain too much if it merged anyway to make sure we have some initial coverage.